SendWithUs.Client/Requests/SendRequestConverter.cs

```diff
@@ -128,21 +141,21 @@ protected void WriteRecipientsList(JsonWriter writer, JsonSerializer serializer,
             foreach (var address in recipients)

```

Comment:
The recipients var is a bunch of strings, each of which is an (email) address. These strings get serialized as a JSON array of the form [ {"address": address1}, {"address": address2} ... ].